### PR TITLE
Task/ASB-747 Address line 3 becomes Town or county

### PR DESCRIPTION
**Pull request by Philip-Philippou1**
Address Line 3 label becomes Town or county in the address entry.  
This commit also fixes a large number of tests due to the click location problem for over 100 tests.

### CODE REVIEW

File: e2e-tests/cypress/support/common-methods/clickProjectLink.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ module.exports = (projectName) => {
 	cy.get('td:nth-child(1)').each(($e1, index) => {
 		const text = $e1.text();
 		if (text.includes(projectName)) {
-			cy.get('td:nth-child(1)').eq(index).click();
+			cy.get('td:nth-child(1)').eq(index).click(15, 20);
```
Comment (JamesP-KC) [inline e2e-tests/cypress/support/common-methods/clickProjectLink.js:5]: I think the issue here is that the query selector is getting the `td` not the actual `a` within it. Clearly up until now, that has worked but i wonder if revising the selector is the "proper" fix.

I did a quick test with this and seemed to work:
```
module.exports = (projectName) => {
	cy.get('td:nth-child(1)').contains(projectName).click();
};
```

**Review by JamesP-KC — APPROVED** [review]

File: e2e-tests/cypress/support/common-methods/clickProjectLink.js (hunk 1 above)
Comment (Philip-Philippou1) [reply e2e-tests/cypress/support/common-methods/clickProjectLink.js]: This works great, will use, thanks

**Review by JamesP-KC — APPROVED** [review]

**Pull request merged**